### PR TITLE
[51211] How it works page - better support for screen readers

### PR DESCRIPTION
**Pull request by jonasofrich**
- Split up locales into multiple strings
- Redone UI on both iOS and Android

Android:
131926915_404122727459008_4523882418225055264_n
132014430_406402270787867_6486382612190748934_n

iOS:
IMG_0187
IMG_0186

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/settings_how_it_works.xml [hunk 1]
```diff
@@ -64,12 +64,132 @@
...
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:gravity="top"
+                        android:lineSpacingExtra="4sp"
+                        style="@style/PrimaryTextRegular"
+                        android:layout_marginBottom="@dimen/defaultVerticalMargin"
+                        android:textAppearance="@style/settings_general"/>
+
+				<TextView
+					    android:id="@+id/settings_how_it_works_heading3"
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19.Droid/Resources/layout/settings_how_it_works.xml:154]: Formatting is wrong here

File: NDB.Covid19/NDB.Covid19/Locales/en.json [hunk 2]
```diff
@@ -73,13 +73,19 @@
...
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_TITLE": "The technology<br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "The app uses Bluetooth and is based on built-in technology developed by Apple and Google. It comprises a system for registering exposure. You can turn this on or off in your phone’s settings.<br><br>Your phone generates a random Bluetooth ID every 15 minutes. When two phones are nearby each other, they exchange IDs.<br><br> Read more about the Smittestopp app at <a href=\"https://www.helsenorge.no/en/smittestopp\">www.helsenorge.no/en/smittestopp</a><br><br>",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_INTRO": "The Smittestopp contact tracing app is a digital initiative in the fight against the spread of COVID-19.",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/en.json:76]: Have you corrected this in the excel sheet?

File: NDB.Covid19/NDB.Covid19/Locales/nb.json [hunk 3]
```diff
@@ -73,13 +73,19 @@
...
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_TITLE": "Teknologien bak<br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "Appen bruker Bluetooth og er basert på en innebygd teknologi som er utviklet av Apple og Google. Teknologien er et system for varsling av smittekontakt. Du kan slå av og på denne varslingen under innstillingene på mobilen din. <br><br>Mobilen din lager en tilfeldig Bluetooth-ID hvert 15. minutt. Når to mobiler er i nærheten av hverandre, utveksler de ID-er.<br><br> Les mer om Smittestopp-appen på <a href=\"https://www.helsenorge.no/smittestopp\">www.helsenorge.no/smittestopp</a><br><br>",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_INTRO": "Smittestopp-appen er et digitalt initiativ i kampen mot spredning av covid-19.",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/nb.json:76]: Corrected in excel sheet?

File: NDB.Covid19/NDB.Covid19/Locales/nn.json [hunk 4]
```diff
@@ -73,13 +73,19 @@
...
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_TITLE": "Teknologien bak<br>",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_3_CONTENT": "Appen nyttar Bluetooth og er basert på ein innebygd teknologi som er utvikla av Apple og Google. Teknologien er eit system for varsling av smittekontakt. Du kan slå av og på denne varslinga under innstillingane på mobilen din.<br><br>Mobilen din lagar ein tilfeldig Bluetooth-ID kvart 15. minutt. Når to mobilar er i nærleiken av kvarandre, utvekslar dei ID-ar.<br><br> Les meir om Smittestopp-appen på <a href=\"https://www.helsenorge.no/smittestopp\">www.helsenorge.no/smittestopp</a><br><br>",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_INTRO": "Smittestopp-appen er eit digitalt initiativ i kampen mot spreiing av covid-19.",
```
Comment (Frederik-S-N-dk) [inline NDB.Covid19/NDB.Covid19/Locales/nn.json:76]: Corrected in excel sheet?

**Review by Frederik-S-N-dk — CHANGES_REQUESTED** [review]

File: NDB.Covid19/NDB.Covid19.Droid/Resources/layout/settings_how_it_works.xml (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19.Droid/Resources/layout/settings_how_it_works.xml]: fixed

File: NDB.Covid19/NDB.Covid19/Locales/en.json (hunk 2 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/Locales/en.json:76]: yes

File: NDB.Covid19/NDB.Covid19/Locales/nb.json (hunk 3 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/Locales/nb.json:76]: yes

File: NDB.Covid19/NDB.Covid19/Locales/nn.json (hunk 4 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/Locales/nn.json:76]: yes

**Review by Frederik-S-N-dk — APPROVED** [review]

**Pull request merged**